### PR TITLE
Make polls respect content warning visibility setting

### PR DESCRIPTION
**Pull request by Tak**
poll-collapsed
poll-expanded

### CODE REVIEW

File: app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java [hunk 1]
```diff
@@ -178,11 +178,13 @@ protected void setSpoilerAndContent(boolean expanded,
                                         @Nullable String spoilerText,
                                         @Nullable Status.Mention[] mentions,
                                         @NonNull List<Emoji> emojis,
+                                        @Nullable PollViewData poll,
                                         final StatusActionListener listener) {
+        setupPoll(poll, emojis, listener);
```
Comment (connyduck) [inline app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java]: This is called even when the poll is currently not visible, right? Why not move it somewhere where its only executed when the poll is actually visible, e.g. into `setTextVisible`?

Comment (Tak) [reply app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java]: I just kept the old code flow as much as possible - I can make this change if you prefer

Comment (connyduck) [reply app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java]: Should be a quick change, so yes please do it

Comment (Tak) [reply app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java]: Updated

**Pull request merged**